dev/breeze/src/airflow_breeze/utils/common_options.py

```diff
@@ -557,6 +557,13 @@ def _set_default_from_parent(ctx: click.core.Context, option: click.core.Option,
     is_flag=True,
     envvar="SKIP_CLEANUP",
 )
+
+option_directory = click.option(
+    "--directory",
+    type=str,
```

Comment:
NIT: This one is better as it allows for auto-completion and nicer error reporting (see what happens when you do not provide it now @amoghrajesh 

```suggestion
    type=click.Path(exists=True, file_okay=False, dir_okay=True, resolve_path=True),
    required=True,
```

Reply:
Makes sense. Handled it :)